app/src/test/java/org/mozilla/focus/ext/AssetManagerTest.kt

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.ext
+
+import android.content.Context
+import android.content.res.AssetManager
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.robolectric.RobolectricTestRunner
+import java.io.ByteArrayInputStream
+
+@RunWith(RobolectricTestRunner::class)
+class AssetManagerTest {
+    /**
+     * Verify that AssetManager.readJSONObject() closes its stream.
+     */
+    @Test
+    fun testReadJSONObjectClosesStream() {
+        // Setup
+
+        val stream = Mockito.spy(ByteArrayInputStream("Hello".toByteArray()))
+
+        val assetManager = Mockito.mock(AssetManager::class.java)
+        Mockito.`when`(assetManager.open(ArgumentMatchers.anyString())).thenReturn(stream)
+
+        val context = Mockito.mock(Context::class.java)
+        Mockito.`when`(context.assets).thenReturn(assetManager)
+
+        // Now use our mock classes to call IOUtils.close()
```

Comment:
nit: IOUtils -> AssetManager (fwiw, I feel like most of the comments here are redundant with the code)

Reply:
Ops, too fast..